### PR TITLE
Add more info to the readme

### PR DESCRIPTION
**Pull request by mkrizek**

### CODE REVIEW

**Comment by mkrizek** [comment]
cc @mattclay

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
 Tools for working with Ansible projects using Azure Pipelines.
+
+## Set up
+1. Install Python dependencies needed to run scripts from this repository to a preferred location using `pip install -r requirements.txt`.
+2. Populate `~/.config/ansible-azp-tools/azure-devops.key` and `~/.config/ansible-azp-tools/github.key` with Personal Access Tokens for Azure Pipelines and Github, respectively.
+3. Run `./sync.py --find --update` to clone the repositories/branches needed for the scripts to check the configurations of the repositories.
+
+## check-pipelines.py
+* `./check-pipelines.py matrix` - checks for out-of-date Azure Pipelines configuration matrix testing against ansible-core devel branch
+* `./check-pipelines.py container` TBD
```
Comment (mattclay) [inline README.md:11]:
```suggestion
* `./check-pipelines.py container` - check for out-of-date `quay.io/ansible/azure-pipelines-test-container` versions
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
 # ansible-azp-tools
 Tools for working with Ansible projects using Azure Pipelines.
+
+## Set up
+1. Install Python dependencies needed to run scripts from this repository to a preferred location using `pip install -r requirements.txt`.
+2. Populate `~/.config/ansible-azp-tools/azure-devops.key` and `~/.config/ansible-azp-tools/github.key` with Personal Access Tokens for Azure Pipelines and Github, respectively.
```
Comment (mattclay) [inline README.md:6]:
```suggestion
2. Populate `~/.config/ansible-azp-tools/azure-devops.key` and `~/.config/ansible-azp-tools/github.key` with Personal Access Tokens for Azure Pipelines and GitHub, respectively.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+3. Run `./sync.py --find --update` to clone the repositories/branches needed for the scripts to check the configurations of the repositories.
+
+## check-pipelines.py
+* `./check-pipelines.py matrix` - checks for out-of-date Azure Pipelines configuration matrix testing against ansible-core devel branch
+* `./check-pipelines.py container` TBD
+
+## pipelines-sanity.py
+TBD
```
Comment (mattclay) [inline README.md:14]:
```suggestion
Run sanity tests against the default branch of each collection in Azure Pipelines
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
...
+
+## pipelines-yaml.py
+TBD
```
Comment (mattclay) [inline README.md:17]:
```suggestion
Output a list of paths or a globs, for finding azure-pipelines.yml files.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
 # ansible-azp-tools
 Tools for working with Ansible projects using Azure Pipelines.
+
+## Set up
+1. Install Python dependencies needed to run scripts from this repository to a preferred location using `pip install -r requirements.txt`.
+2. Populate `~/.config/ansible-azp-tools/azure-devops.key` and `~/.config/ansible-azp-tools/github.key` with Personal Access Tokens for Azure Pipelines and Github, respectively.
+3. Run `./sync.py --find --update` to clone the repositories/branches needed for the scripts to check the configurations of the repositories.
+
+## check-pipelines.py
+* `./check-pipelines.py matrix` - checks for out-of-date Azure Pipelines configuration matrix testing against ansible-core devel branch
```
Comment (mattclay) [inline README.md:10]:
```suggestion
Check for out-of-date Azure Pipelines configurations testing against ansible-core.

* `./check-pipelines.py matrix` - check for out-of-date platform references in the test matrix
```

**Review by mattclay — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,2 +1,17 @@
 # ansible-azp-tools
 Tools for working with Ansible projects using Azure Pipelines.
+
+## Set up
```
Comment (mattclay) [inline README.md:4]:
```suggestion
## Set up

NOTE: Python 3.10 or 3.11 are required. The Azure DevOps SDK does not yet support Python 3.12.
```

**Review by mattclay — APPROVED** [review]

**Pull request merged**